<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SettingsView</class>
 <widget class="QWidget" name="SettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>542</width>
    <height>368</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="styleSheet">
      <string notr="true">font: 70pt &quot;ArcadeClassic&quot;;</string>
     </property>
     <property name="text">
      <string>Settings</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignCenter</set>
     </property>
    </widget>
   </item>
   <item row="7" column="0">
    <widget class="QPushButton" name="backButton">
     <property name="styleSheet">
      <string notr="true">padding: 15px;
background-color: Tomato;
font: 25pt &quot;ArcadeClassic&quot;;</string>
     </property>
     <property name="text">
      <string>Back</string>
     </property>
    </widget>
   </item>
   <item row="6" column="0">
    <spacer name="verticalSpacer_2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="1" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="2" column="0">
    <widget class="QPushButton" name="musicButton">
     <property name="styleSheet">
      <string notr="true">background-color: SteelBlue;
padding: 15px;
font: 25pt &quot;ArcadeClassic&quot;;</string>
     </property>
     <property name="text">
      <string>Music : no</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="5" column="0">
    <widget class="QPushButton" name="soundButton">
     <property name="styleSheet">
      <string notr="true">background-color: SteelBlue;
padding: 15px;
font: 25pt &quot;ArcadeClassic&quot;;</string>
     </property>
     <property name="text">
      <string>Sound : no</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
